aptos-move/block-executor/src/view.rs

```diff
@@ -13,7 +13,9 @@ use aptos_aggregator::{
 };
 use aptos_logger::error;
 use aptos_mvhashmap::{
-    types::{MVDataError, MVDataOutput, MVModulesError, MVModulesOutput, TxnIndex},
+    types::{
+        MVAggregatorsError, MVDataError, MVDataOutput, MVModulesError, MVModulesOutput, TxnIndex,
+    },
```

Comment:
Generally speaking, is there any advantage in using so many Error types?

Reply:
These errors come from different mvhashmaps, and having them with different types seems cleaner in this case.